interpreter/terminal_interface/start_terminal_interface.py

```diff
@@ -541,11 +540,13 @@ def start_terminal_interface(interpreter):
         #         print(chunk.get("content"))
 
         # Give it access to the computer via Python
-        interpreter.computer.run(
-            "python",
-            "import time\nfrom interpreter import interpreter\ncomputer = interpreter.computer",  # We ask it to use time, so
+        for _ in interpreter.computer.run(
```

Comment:
This is actually why I added that part that pulls from the stream! It allows you to just run `interpreter.computer.run` without the `for _ in` syntax. I'll merge but will revert this.